config/routes.rb

```diff
@@ -38,6 +38,9 @@
     get 'app_configuration/redigera', to: 'app_configuration#edit',
         as: :edit_app_configuration
 
+    get 'user_profile_edit/:id', to: 'user_profile#edit', as: :user_profile_edit
```

Comment:
I know we're not entirely consistent with the route naming, but in general we have used Swedish words.  So using Swedish and following the standard Rails routing patterns (and how we're generally doing the paths), these should be:
```
get 'anvandare/:id/redigera, to: 'user_profile#edit', as: :user_profile_edit
put 'anvandare/:id', to: 'user_profile#update', as: :user_profile_update
get 'anvandare/:id/<swedish word for "become">', to: 'user_profile#become', as: :become_user
```

Reply:
All of the "admin" routes seem to be have English URL's.  The user-visible routes seem to all(?) in Swedish, which is good.

I'd like to propose we make this kind of change through another story - of we decide that we need to adhere to the same standard for admin actions (which a user/member should never see).

Reply:
Agree to have the route-stuff in another story. One or more. :)

Reply:
agreed - another story is fine.